# set pose of the robot
# theta(angular position)
# x/y (linear position)
# set_pose_state(true if new pose service is activated)
float32 theta
float32 x
float32 y
---
bool set_pose_state